KarhooSDK/Api/Response/CancellationFeePrice.swift

```diff
@@ -33,6 +36,7 @@ public struct CancellationFeePrice: KarhooCodableModel {
         self.currency = (try? container.decode(String.self, forKey: .currency)) ?? ""
         self.type = (try? container.decode(String.self, forKey: .type)) ?? ""
         self.value = (try? container.decode(Int.self, forKey: .value)) ?? 0
+        self.decimalValue = Double(value) * 0.01
```

Comment:
I would suggest making 0.01 as a `const` to avoid magic number withing the logic.

Reply:
Done. Also update QuotePrice to use it.